go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/trento-project/agent
 
-go 1.23
+go 1.23.4
+
+toolchain go1.23.6
```

Comment:
I don't honestly know why the `go get` and `go tidy` are putting this. We can remove them manually though

Reply:
See no evil